.travis.yml

```diff
@@ -1,97 +0,0 @@
-language: node_js
-
-# 2*2 matrix for tests
-os:
-  - linux
-  - windows
-node_js:
-  - 10
-  - 12
-
-jobs:
-  include:
-    # run tests in Node.js stable and publish code coverage
-    - name: 'Coverage'
-      os: linux
-      node_js: stable
-      script:
-        - npm run test-unit
-        - npm run codecov -- -c -Z -f .coverage/coverage-final.json -F unit
-
-    # create a new release if $CREATE_RELEASE is set
-    - stage: Create Release
```

Comment:
@codenirvana Should we also add release and publish Actions? I'll add them as separate actions instead of putting in the `ci.yml`.